README.md

```diff
@@ -1,67 +1,91 @@
-
-### Standard Library
-
-Aurae exposes its functionality over a gRPC API which is referred to as the [Aurae Standard Library](https://aurae.io/stdlib/).
-
-### Principle of Least Awareness
-
-A single Aurae instance has no awareness of higher order scheduling mechanisms such as the Kubernetes control plane. Aurae is designed to take ownership of a single node, and expose the standard library as a generic and meaningful way for higher order consumers.
-
-Aurae is a low level building block and is designed to work well with any higher order system by offering a thoughtful set of APIs and controls for managing workloads on a node.
-
-### Motivation
-
-Read [Why fix Kubernetes and Systemd](https://medium.com/@kris-nova/why-fix-kubernetes-and-systemd-782840e50104) by [Kris Nóva](https://github.com/krisnova).
-
-Aurae attempts to simplify and improve the stack in enterprise distributed systems by carving out a small portion of responsibility while offering a few basic guarantees with regard to state, synchronicity, awareness, and security.
-
-Aurae brings enterprise identity as low as the socket layer in a system, which unlocks multi tenant workloads that run below tools like Kubernetes.
-
-## AuraeScript
-
-Aurae offers a Turing complete scripting language built on top of TypeScript called [AuraeScript](https://github.com/aurae-runtime/aurae/tree/main/auraescript). AuraeScript embeds the [Deno](https://deno.land) source code directly, and offers a remote client and SDK to interface directly with Aurae remotely. The AuraeScript library is automatically generated from the `.proto` files defined in the [Aurae Standard Library](https://aurae.io/stdlib/).
-
-Valid TypeScript files can be leveraged to replace static manifests, such as YAML, as well as interact directly with a running system.
+<!-- THE DOCUMENT -->
+
+![Deploy (aurae.io)] ![Documentation] ![Main Build (lint, compile, test)]
+
+# Summary
+
+[Aurae] is a free and [open-source] project built with the [Rust] systems
+language.
+
+The project adapts to the expanding demands of enterprise distributed systems
+that meets the present and future demands for brilliant higher-order distributed
+systems.
+
+Aurae supports multi-tenant workloads and enterprise identities all the way down
+to the socket layer and can rationalize workloads and further improve system
+proportions by utilizing a variety of isolation strategies for enterprise
+workloads, the project can work exceptionally well with tools like Kubernetes [^compare].
+
+## Project Status
+
+> **STILL IN EARLY DEVELOPMENT!**<br>
+> **The Aurae project and API can change without notice.**<br> 
+> **Do not run the project in production until further notice!**
+> <br><br>
+> _"Aurae attempts to simplify and improve the stack in_
```

Comment:
I suggest getting rid of my quote here. I don't need to be (nor do I want to be) the "star of the show". We can have the quote, just maybe get it off the main repo.

Reply:
Pardon me. For that one.